data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
+  "shelfmark": "Sinai Syriac 2A",
+  "collection": "Old Collection",
+  "repository": "https://sinaimanuscripts.library.ucla.edu/place/1",
+  "form": "Codex",
+  "binding": {
+    "exists": true,
+    "type": "",
+    "note": ["Relatively recent; leather on wood showing signs of repair; metal studs (some missing) on front and back and along edges; remnants of clasps (four total, two for edge and one each at top and bottom)"],
+    "features": []
```

Comment:
we should categorize the list of features - then see if we have any binding-related ones

Reply:
JIRA ticket for this: https://jira.library.ucla.edu/browse/SLDP-632